operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +185,28 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Responsibilities
+
+### Identifying and ranking gaps and pain points among customers
+
+When it comes to identifying gaps and pain points, ranking them is important in order to correctly prioritize.
+
+#### Identification
+
+- Monitor support tickets and get regular feedback from Support Engineers through the support channel
+- Ensure information flow with Solutions Architects and Customer Success Managers e.g. by receiving a regular report
```

Comment:
```suggestion
- Ensure information flow with Solution Architects and Customer Success Managers (e.g. by receiving a regular report).
```
Should we elaborate on the type of report, or provide a few more examples of information flow mechanisms?